package.json

```diff
@@ -15,8 +15,8 @@
     "format": "prettier --write 'src/**/*.ts?(x)' && npm run lint -- --fix",
     "typecheck": "tsc --noEmit",
     "clean": "rm -rf ./dist ./tmp",
-    "build": "npm run clean && rollup --config && mv tmp/*.js dist",
-    "prepack": "npm run build",
+    "build": "npm run clean && rollup --config && mv tmp/*.js dist && attw --pack .",
+    "prepublish": "npm run build",
```

Comment:
Moved to prepublish, since attw --pack runs npm pack, creating an infinite loop with prepack